src/Altinn.App.Core/Helpers/IDataModel.cs

```diff
@@ -19,17 +16,17 @@ public interface IDataModelAccessor
     /// "Bedrifter[1].Ansatte.Alder", will fail, because the indicies will be reset
     /// after an inline index is used
     /// </remarks>
-    object? GetModelData(string key, ReadOnlySpan<int> indicies = default);
+    object? GetModelData(ModelBinding key, ReadOnlySpan<int> indicies = default);
```

Comment:
Another tricky one in terms of not making breaking changes. Someone could have implemented this interface and will get a broken build for this. I don't think anyone has though (couldn't find anything when searching)